app/src/main/java/com/lucas/medicaltools/MainActivity.kt

```diff
@@ -1,11 +1,76 @@
 package com.lucas.medicaltools
 
+import android.annotation.SuppressLint
+import android.app.SearchManager
+import android.content.Context
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.Menu
+import android.view.View
+import android.widget.ProgressBar
+import androidx.appcompat.widget.SearchView
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.lucas.medical_equip.repository.MedicalTool
+import com.lucas.medical_equip.service.RetrofitBuilder
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.schedulers.Schedulers
+import mu.KotlinLogging
 
+private val logger = KotlinLogging.logger {}
 class MainActivity : AppCompatActivity() {
+    private lateinit var medicalTools: List<MedicalTool>
+    private lateinit var medicalRecyclerView: RecyclerView
+    private lateinit var medicalAdapter: MedicalToolAdapter
+    private lateinit var progressBar: ProgressBar
+    private lateinit var searchView: SearchView
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        progressBar = findViewById(R.id.progressBar1)
```

Comment:
If you use Kotlin binding, you don't need to use findViewById, you can just reference the item by the id.